_migration-assistant/migration-phases/index.md

```diff
@@ -7,7 +7,7 @@ has_children: true
 
 This page details how to conduct a migration with Migration Assistant. It encompasses a variety of scenarios including:
 
-- [**Metadata migration**]({{site.url}}{{site.baseurl}}/migrations/migration-phases/migrating-metadata/): Migrating cluster metadata, such as index settings, aliases, and templates.
-- [**Backfill migration**]({{site.url}}{{site.baseurl}}/migrations/migration-phases/backfill/): Migrating existing or historical data from a source to a target cluster.
+- [**Metadata migration**]({{site.url}}{{site.baseurl}}/migration-assistant/migration-phases/migrating-metadata/): Migrating cluster metadata, such as index settings, aliases, and templates.
+- [**Backfill migration**]({{site.url}}{{site.baseurl}}/migration-assistant/migration-phases/backfill/): Migrating existing or historical data from a source to a target cluster.
```

Comment:
Can't comment on the actual file because it seems already merged but in `migrating-metadata.md`, the example in the details block is not displaying correctly.

Reply:
I'll remove the details HTML. Left over from an initial draft.